models/cxr_image_synthesis_latent_diffusion_model/configs/inference.json

```
@@ -0,0 +1,115 @@
+{
+    "imports": [
+        "$import torch",
+        "$from datetime import datetime",
+        "$from pathlib import Path",
+        "$from transformers import CLIPTextModel",
+        "$from transformers import CLIPTokenizer"
+    ],
+    "bundle_root": ".",
+    "dataset_dir": "",
+    "dataset": "",
+    "evaluator": "",
+    "inferer": "",
+    "load_old": 1,
+    "model_dir": "$@bundle_root + '/models'",
+    "output_dir": "$@bundle_root + '/output'",
+    "create_output_dir": "$Path(@output_dir).mkdir(exist_ok=True)",
+    "prompt": "Big right-sided pleural effusion",
+    "prompt_list": "$['', @prompt]",
+    "guidance_scale": 7.0,
+    "device": "$torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')",
+    "tokenizer": "$CLIPTokenizer.from_pretrained(\"stabilityai/stable-diffusion-2-1-base\", subfolder=\"tokenizer\")",
+    "text_encoder": "$CLIPTextModel.from_pretrained(\"stabilityai/stable-diffusion-2-1-base\", subfolder=\"text_encoder\")",
+    "tokenized_prompt": "$@tokenizer(@prompt_list, padding=\"max_length\", max_length=@tokenizer.model_max_length, truncation=True,return_tensors=\"pt\")",
+    "prompt_embeds": "$@text_encoder(@tokenized_prompt.input_ids.squeeze(1))[0].to(@device)",
+    "out_file": "$datetime.now().strftime('sample_%H%M%S_%d%m%Y')",
+    "autoencoder_def": {
+        "_target_": "monai.networks.nets.AutoencoderKL",
+        "spatial_dims": 2,
+        "in_channels": 1,
+        "out_channels": 1,
+        "latent_channels": 3,
+        "channels": [
+            64,
+            128,
+            128,
+            128
+        ],
+        "num_res_blocks": 2,
+        "norm_num_groups": 32,
+        "norm_eps": 1e-06,
+        "attention_levels": [
+            false,
+            false,
+            false,
+            false
+        ],
+        "with_encoder_nonlocal_attn": false,
+        "with_decoder_nonlocal_attn": false
+    },
+    "network_def": "@autoencoder_def",
```

```suggestion
    "network_def": "@diffusion_def",
```
If the "main" network for this bundle is the diffusion model then this should be referred to here, though I don't know if it really matters for correct functioning of the bundle.